json/UI_Weaponoid_Name.txt

```diff
@@ -772,7 +772,7 @@
 	{
 		"assign": "131",
 		"jp_text": "永劫無極の絶対剣 オロチアギト",
-		"tr_text": "The Eternal Blade Orochiagito"
+		"tr_text": "The Eternal Blade Orochi Agito"
 	},
```

Comment:
Leave it as one word, making a redaction of the weapon to be `Orochiagito` instead.